perftracker/templates/job_id_edit.html

```diff
@@ -0,0 +1,134 @@
+     aria-hidden="true">
+    <div class="modal-dialog modal-sm">
+        <div class="modal-content">
+
+            <button class="btn btn-primary btn-lg" id="btn_show_job_dialog" data-toggle="modal"
+                    data-target="#JobDialog"></button>
+
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <h4 class="modal-title" id="jobLabel"></h4>
+            </div>
+
+            <div class="modal-body">
+                <form id="jobForm">
+                    <div class="form-group">
+                        <label for="Product name">Product name</label>
+                        <input type="hidden" id="jobId">
+                        <input type="product_name" class="form-control" id="productName" maxlength="128"
+                               placeholder="Product Name" value="">
+                    </div>
+
+                    <div class="form-group">
+                        <label for="Product ver">Product ver</label>
+                        <input type="hidden" id="jobId">
+                        <input type="product_ver" class="form-control" id="productVer" maxlength="128"
+                               placeholder="Product Ver" value="">
+                    </div>
+
+                    <div class="form-group">
+                        <label for="Job Title">Title</label>
+                        <input type="hidden" id="jobId">
+                        <input type="title" class="form-control" id="jobTitle" maxlength="512" placeholder="Job Title"
+                               value="">
+                    </div>
+
+                    <div class="form-group">
+                        <label for="Suite ver">Test ver</label>
+                        <input type="hidden" id="jobId">
+                        <input type="suite_ver" class="form-control" id="suiteVer" maxlength="128"
+                               placeholder="Suite Ver" value="">
+                    </div>
+
+                    <div class="form-group">
+                        <label for="Duration">Duration</label>
+                        <input type="hidden" id="jobId">
+                        <input type="duration" class="form-control" id="duration" maxlength="128" placeholder="Duration"
+                               value="">
+                    </div>
+
+                    <div class="form-group">
+                        <label for="Links">Links</label>
+                        <input type="hidden" id="jobId">
+                        <textarea type="links" class="form-control" id="links" maxlength="1024" placeholder="Links"
+                                  value=""></textarea>
+                    </div>
+
+                    <div class="form-group">
+                        <label for="UUID">UUID</label>
```

Comment:
I think only user-facing data must be edited. UUID mustn't be editable because it is "primary key" of a job. For example, a user can change the unique id to non-unique and it will cause issues somewhere. Or results re-upload from a client will create another job because it will not find needed UUID, etc.

Reply:
Fixed